rewarderConfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "LPToken": "0x044b6b0cd3bb13d2b9057781df4459c66781dce7",
+  "RewardToken": "0x09c9d464b58d96837f8d8b6f4d9fe4ad408d3a4f",
+  "CoingeckoRewarderTokenName": "Aurigami",
```

Comment:
@evienft this should be aurigami (lower case)

https://www.coingecko.com/en/coins/aurigami -- see the "API ID", copy paste that in
Remove the comma after aurigami as well